smk/dependencies.smk

```diff
@@ -501,6 +501,12 @@ rule motus_db:
             if [ $? -eq 0 ]; then
                 echo 'mOTUs3 database download: OK'
                 echo OK > {output}
+
+                # place db into data folder
+                $MOTUS_DB_PATH=$(find "$(dirname $(which motus))/../lib" -type d -iname db_mOTU | head -1 )
+                rsync -a $MOTUS_DB_PATH {minto_dir}/data/motus/{motus_version}/
```

Comment:
Does this work with a fresh installation? I cannot see where `motus_db_path` is created. And it is not part of an output either (because then Snakemake would have created it). For a fresh installation, it would always end up with `motus_db_path = ""` and this line might fail since the directory does not exist?

Reply:
I only tested it with 1 level of dirs missing, and rsync can handle that, but not more. Fixed in new commit.

Reply:
Oh you meant `motus_db_path` in QC_2 - if the folder doesn't exists, then it should fall back to using the default database location in the python package.